### PR TITLE
[PHP] better operationId handling, add CI for PHP slim server

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [ ] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.0.x`. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

- better operationId handling (for operationId starting with a number)
- add CI for PHP slim server

### CODE REVIEW

**Comment by wing328** [comment]
I've created "php_syntax_checker.bash", which can be reused for other PHP server samples (e.g. lumen)

cc @jebentier (2017/07) @dkarlovi (2017/07) @mandrean (2017/08) @jfastnacht (2017/09) @ackintosh (2017/09) @ybelenko (2018/07)

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java [hunk 1]
```diff
@@ -496,6 +496,12 @@ public String toOperationId(String operationId) {
             operationId = "call_" + operationId;
         }
 
+	// operationId starts with a number
```
Comment (ackintosh) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java]: please replace tabs with spaces.

File: samples/server/petstore/php-slim/pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                <version>1.2.1</version>
+                <executions>
+                    <execution>
+                        <id>syntax-check</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+			    <executable>./php_syntax_checker.bash</executable>
```
Comment (ackintosh) [inline samples/server/petstore/php-slim/pom.xml]: please replace tabs with spaces.

Comment (wing328) [reply samples/server/petstore/php-slim/pom.xml]: 👍 Yup, noticed that as well. Done via https://github.com/OpenAPITools/openapi-generator/pull/723/commits/9185150c0a0122bfe5ff1af2a9c759c41212d1da

Comment (wing328) [reply samples/server/petstore/php-slim/pom.xml]: Done via f83df38. Also updated vim setting to use spaces instead of tabs.

**Pull request merged**